libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -32,6 +33,21 @@ export const removeFromReadingList = createAction(
   props<{ item: ReadingListItem }>()
 );
 
+export const markedAsFinished = createAction(
+  '[Reading List] Marked as finished',
+  props<{ item: ReadingListItem }>()
+);
+
+export const markFinshedSuccess = createAction(
+  '[Reading List] Book finished',
+  props<{ item: Update<ReadingListItem> }>()
+);
+
+export const markFinishFailed = createAction(
+  '[Reading List] Mark as finished failed',
```

Comment:
Same goes for `markFinishFailed`. For consistency I would call it `failedMarkAsFinished` and the text `[Reading List] Failed mark as finished`